### ISSUE TITLE
Pre-Process GS

### ISSUE THREAD

**Issue by nikita-jaya** [gradebook-dev/gradebook#58]
How to pre-process gs
- Maybe shift to email
- Use unique id-cols, etc
- Will be in `get_grades()`

**Comment by mvoutov** [comment]
Are there any items from this list that still need work? @nikita-jaya

**Comment by nikita-jaya** [comment]
> Are there any items from this list that still need work? [@nikita-jaya](https://github.com/nikita-jaya)

Yep! I don't remember how we deal with students with NA SIDs, but I don't remember if merging duplicate students is now dealt with on the Gradescope end or not
